From 72c38010241d97bf595e8eabfa81da2b297dd082 Mon Sep 17 00:00:00 2001
From: Stefano Babic <sbabic@denx.de>
Date: Wed, 27 Jul 2016 15:45:13 +0200
Subject: [PATCH 2/3] suricatta: updated to new U-Boot Environment API

Signed-off-by: Stefano Babic <sbabic@denx.de>
---
 suricatta/state.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/suricatta/state.c b/suricatta/state.c
index 434f8e4..0065f85 100644
--- a/suricatta/state.c
+++ b/suricatta/state.c
@@ -49,7 +49,7 @@ server_op_res_t reset_state(char *key)
 #ifdef CONFIG_SURICATTA_STATE_CHOICE_UBOOT
 server_op_res_t save_state(char *key, update_state_t value)
 {
-	if (fw_env_open() != 0) {
+	if (fw_env_open(NULL) != 0) {
 		ERROR("Error: Cannot initialize U-Boot environment.\n");
 		return SERVER_EERR;
 	}
@@ -57,12 +57,12 @@ server_op_res_t save_state(char *key, update_state_t value)
 		ERROR("Error: Cannot write to U-Boot's environment.\n");
 		return SERVER_EERR;
 	}
-	return fw_env_close() == 0 ? SERVER_OK : SERVER_EERR;
+	return fw_env_close(NULL) == 0 ? SERVER_OK : SERVER_EERR;
 }
 
 server_op_res_t read_state(char *key, update_state_t *value)
 {
-	if (fw_env_open() != 0) {
+	if (fw_env_open(NULL) != 0) {
 		ERROR("Error: Cannot initialize U-Boot environment.\n");
 		return SERVER_EERR;
 	}
@@ -79,7 +79,7 @@ server_op_res_t read_state(char *key, update_state_t *value)
 }
 server_op_res_t reset_state(char *key)
 {
-	if (fw_env_open() != 0) {
+	if (fw_env_open(NULL) != 0) {
 		ERROR("Error: Cannot initialize U-Boot environment.\n");
 		return SERVER_EERR;
 	}
@@ -87,6 +87,6 @@ server_op_res_t reset_state(char *key)
 		ERROR("Error: Cannot write to U-Boot's environment.\n");
 		return SERVER_EERR;
 	}
-	return fw_env_close() == 0 ? SERVER_OK : SERVER_EERR;
+	return fw_env_close(NULL) == 0 ? SERVER_OK : SERVER_EERR;
 }
 #endif /* CONFIG_SURICATTA_STATE_CHOICE_UBOOT */
-- 
2.7.4

